poetry/core/pyproject/__init__.py

```diff
@@ -1,6 +0,0 @@
-from poetry.core.pyproject.exceptions import PyProjectException
```

Comment:
This removal breaks https://github.com/python-poetry/poetry/blob/6eaa2984d9fb51a837c4ab11643f4dd9d7828d4d/poetry/console/commands/config.py#L14 (current master).